nullaway/src/main/java/com/uber/nullaway/dataflow/AccessPath.java

```diff
@@ -255,6 +256,38 @@ public static AccessPath getAccessPathForNodeWithMapGet(Node node, @Nullable Typ
     }
   }
 
+  /**
+   * Gets corresponding AccessPath for a class field element.
+   *
+   * @param field The class field element.
+   * @param receivers The list of receivers.
+   * @return corresponding AccessPath.
+   */
+  public static AccessPath fromFieldAndBase(Element field, List<Element> receivers) {
+    List<AccessPathElement> elements = new ArrayList<>();
+    elements.add(new AccessPathElement(field));
+    Root root;
+    Element rootReceiver;
+    if (receivers == null || receivers.size() == 0) {
+      root = new Root();
+    } else {
+      rootReceiver = receivers.get(0);
+      assert !(rootReceiver instanceof MemberSelectTree);
+      if (rootReceiver == null
```

Comment:
Why do we allow `null` elements in the `receivers` list?  I think we should expect everything in the list to not be null